kahuna/public/js/components/gr-add-label/gr-add-label.css

```diff
@@ -6,3 +15,15 @@ gr-add-label .small {
     border: none;
     padding: 0;
 }
+
+gr-add-label form {
+    display: flex;
+}
+
+.gr-add-label__buttons {
+    display: flex;
+}
+
+.gr-add-label__buttons button{
```

Comment:
Same here - in case we want to add another button that shouldn't have this styling.